modules/identity/src/Volo.Abp.Identity.Application/Volo/Abp/Identity/IdentityRoleAppService.cs

```diff
@@ -69,7 +69,7 @@ public async Task UpdatePermissionsAsync(Guid id, UpdatePermissionsDto input)
         [Authorize(IdentityPermissions.Roles.Create)]
         public async Task<IdentityRoleDto> CreateAsync(IdentityRoleCreateDto input)
         {
-            var role = new IdentityRole(GuidGenerator.Create(), input.Name, CurrentTenant.Id);
+            var role = new IdentityRole(GuidGenerator.Create(), input.Name, input.IsDefault, false, input.IsPublic, CurrentTenant.Id);
```

Comment:
Remove `input.IsDefault, false, input.IsPublic` from constructor and set using public setters. Because setting these properties are not mandatory to create the object. Constructor should only get minimal required properties.